packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -22,21 +23,25 @@ export const baseUrlFromChainId = (chainId: string): string => {
       return 'https://avalanche.api.0x.org/'
     case KnownChainIds.OptimismMainnet:
       return 'https://optimism.api.0x.org/'
+    case KnownChainIds.BnbSmartChainMainnet:
```

Comment:
Since they're all subdomains of `api.0x.org`, we may want to make this programmatic but realistically doesn't matter at all

Reply:
yeah not worth at this point imo, going to leave as is